components/compositing/compositor.rs

```diff
@@ -375,8 +378,6 @@ impl<Window: WindowMethods + ?Sized> IOCompositor<Window> {
             shutdown_state: ShutdownState::NotShuttingDown,
             page_zoom: Scale::new(1.0),
             viewport_zoom: PinchZoomFactor::new(1.0),
-            min_viewport_zoom: None,
-            max_viewport_zoom: None,
```

Comment:
I am using the max_viewport_zoom in the android port to clamp the maximium value of MAX_ZOOM (currently used for page zoom). Is it not useful to have this clamping in compositor, or is that something we expect the layers above to handle (in this case the android event handling logic in Java)

Reply:
No, it probably make sense to have this in the compositor.

Reply:
I've restored these members, but set the default min zoom to 1.0.